src/app/util/attribute-storage.cpp

```diff
@@ -884,7 +884,7 @@ static uint16_t findClusterEndpointIndex(EndpointId endpoint, ClusterId clusterI
 
     if (emberAfFindClusterWithMfgCode(endpoint, clusterId, mask, manufacturerCode) == NULL)
     {
-        return 0xFF;
+        return 0xFFFF;
```

Comment:
We really should have added symbolic constants for all this stuff.  :(  @vivien-apple followup?

Reply:
@bzbarsky-apple That’s in my todo list!